content/blog/integrating-devops-and-security-for-scalable-platform-engineering/index.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Integrating DevOps and Security for Platform Engineering"
+allow_long_title: true
+
+# The date represents the post's publish date, and by default corresponds with
+# the date and time this file was generated. Dates are used for display and
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2024-12-05T07:41:06Z
+
+# The draft setting determines whether a post is published. Set it to true if
+# you want to be able to merge the post without publishing it.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc: Gain insights from industry leaders on integrating DevOps and security in platform engineering to build scalable, secure, and developer-friendly platforms.
+
+# The meta_image appears in social-media previews and on the blog home page. A
+# placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the
+# `id` properties of the team member files at /data/team/team. Create a file for
+# yourself if you don't already have one.
+authors:
+    - sara-huddleston
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
```

Comment:
tags should really just be used for navigational purposes; i.e., does this help the reader navigate our blog to find other relevant content? will we produce other content under that tag (e.g. devex)?

Reply:
Yep, it shows all the content with the same tag. Although, being at the very bottom isn't as useful, but it can make discovery of content with the same topic = tags easier.